rhv_checks.py

```diff
@@ -338,6 +343,48 @@ def check_vms_distributed_hosts(system, warn=5, crit=10):
         sys.exit(3)
 
 
+def check_services_running(system):
+    """Check to see if the vdsm, ovirt-ha-agent, ovirt-ha-broker service is up/running"""
+    hosts = system.api.system_service().hosts_service()
+    hosts_agents = dict()
+    hosts_status = dict()
+    for host in hosts.list():
+        host_service = hosts.host_service(host.id)
+        ip_addresses = host_ips(host_service)
+        ssh = paramiko.SSHClient()
```

Comment:
I would like to have `SSH` stuff outside of `rhv_checks`, preferably in the `utils.py`. May be other tests requires it.

Reply:
thanks, makes sense

Reply:
@lina-nikiforova you can refer this PR --> https://github.com/kedark3/check-rhv/pull/5